components/navBar/NavBarMenuButton.tsx

```diff
@@ -1,4 +1,4 @@
-const NavBarMenuButton = () => {
+function NavBarMenuButton() {
   return (
     <svg
```

Comment:
Place this inside an SVG and import only the SVG if possible.

Reply:
![Screenshot 2023-09-15 at 2 37 25 AM](https://github.com/DevTaehong/portfolio_jsm/assets/71358207/0b11669d-e0c4-41c3-83c5-fbb70e179e12)

I'm using it as an icon for NexUI and the icon prop should be ReactNode.